src/utils/constants/landing.ts

```diff
@@ -7,76 +7,76 @@ export const MENU_SECTIONS = [
       "link": "https://www.linkedin.com/in/roberto-reyes-fragoso/"
   },
   {
       "name": "Andrés Rodarte",
       "text": "I am a Computer Systems Engineering Student from México. I like to develop web apps for all kinds of solutions from games to personal blogs.",
-      "avatar": "https://media-exp1.licdn.com/dms/image/C5603AQGGQxOAvVUIxg/profile-displayphoto-shrink_800_800/0/1620963319318?e=1626912000&v=beta&t=C2FhUvlokKNuLAsj_9bxqmsYZsE7HN-1JeEUFnxlgcw",
+      "avatar": "images/Collaborators/Andres.jpeg",
       "link": "https://www.linkedin.com/in/andr%C3%A9s-rodarte-l%C3%B3pez/"
   },
   {
       "name": "Duhart Jiménez",
       "text": "",
-      "avatar": "https://media-exp1.licdn.com/dms/image/C4E03AQE77RphSZuEXg/profile-displayphoto-shrink_800_800/0/1598132634291?e=1626912000&v=beta&t=9z8-ZmhVPla4lTd7S6PkGePHmRWrAqlFtjUXnOVFEL4",
+      "avatar": "images/Collaborators/Duhart.jpeg",
       "link": "https://www.linkedin.com/in/vhjd"
   },
   {
       "name": "Carlos Paredes",
       "text": "Computer Systems Engineering student that likes helping others build their programming and social skills throught many growing student-based ogranizations. Aiming to be a global life changing factor building software that everyone can use regardless their capabilities.",
-      "avatar": "https://media-exp1.licdn.com/dms/image/C4E03AQFx8xWkMkX2pw/profile-displayphoto-shrink_800_800/0/1613021671072?e=1626912000&v=beta&t=8x5unH57rt7TiX4MuNtvRyEmQdW5YeBfDP4_oGrDiis",
+      "avatar": "images/Collaborators/Carlos.jpeg",
       "link": "https://www.linkedin.com/in/carlosparedescid"
   },
   {
       "name": "Óscar Rosas",
       "text": "Computer Science student at UNAM | Computer Systems at ESCOM IPN. Former Software Engineer Intern at Facebook; Incoming SWE intern at Microsoft.",
-      "avatar": "https://media-exp1.licdn.com/dms/image/C4E03AQHRwtSCl3rb2w/profile-displayphoto-shrink_800_800/0/1619201646943?e=1626912000&v=beta&t=xyqi5_Kk2RGY5BOYapPAeE4nMnKterQO230p8C1fvzM",
+      "avatar": "images/Collaborators/Oscar.jpeg",
       "link": "https://www.linkedin.com/in/soyoscarrh"
   },
   {
       "name": "Daniela Vera",
       "text": "",
-      "avatar": "https://media-exp1.licdn.com/dms/image/C5603AQHyX3lRB8Ijzg/profile-displayphoto-shrink_800_800/0/1613790705206?e=1626912000&v=beta&t=443BH5A6LT0JcIlpv1QXiemlVsvLRtLCGVhKdmTNKhw",
+      "avatar": "images/Collaborators/Daniela.jpeg",
       "link": "https://www.linkedin.com/in/daniela-vera-garc%C3%ADa-70766a207/"
   },
-  {
-      "name": "Alberto Esquivel",
-      "text": "",
-      "avatar": "https://scontent-dfw5-1.xx.fbcdn.net/v/t1.6435-9/72650239_2495946783785482_1474312971063656448_n.jpg?_nc_cat=103&ccb=1-3&_nc_sid=09cbfe&_nc_eui2=AeG8-MH7zHGb6CEnoaJjAY4poOIoHGAGhNmg4igcYAaE2cqmQwUiufbUdsUR1DQwxhc&_nc_ohc=sHg2kBk3-mUAX-LY6aK&_nc_ht=scontent-dfw5-1.xx&oh=8a470b010ae60f0903661eceeb077427&oe=60C91ACA",
-      "link": "https://www.linkedin.com/in/aesquivelv/"
-  },
```

Comment:
Why did you erase Alberto?

Reply:
I moved him to the top, is not deleted, He doesn't have any profile picture. or at least I couldn't find any and moving it to the top makes him visible only if people scroll all the way to the left.

Reply:
Oh I didn't see that, makes sense!

Reply:
Also, if you could help finding images for the missing people , I'll appreciate it